architecture/azure/processes.adoc

```diff
@@ -0,0 +1,51 @@
+[[azure-processes]]
+= Processes
+//openshift-aro 3.11 specific
+
+This section presents the summary of how Azure Red Hat OpenShift specific
+processes work. These descriptions do not go into all edge cases, and are
+limited to presenting the expected flow. Edge case handling can be observed
+in the source code repository for Azure Red Hat OpenShift.
+
+== Cluster update
+A new release can contain new Virtual Machine images or container images.
+It is done in the background without any customer involvement or any
+interruption to customer applications running on the cluster.
+All actions are done by the Azure Red Hat OpenShift Resource Provider.
+
+=== Preparation
+
+After the resource provider switches the cluster into the appropriate
+provisioning state it performs a backup of Etcd. Then generates all required
```

Comment:
s/Then generates/The process continues by generating/
s/places/placing/
s/the/their/